packages/template-retail-react-app/app/pages/checkout/partials/contact-info.jsx

```diff
@@ -183,8 +185,8 @@ const ContactInfo = () => {
                     onClose={() => setSignOutConfirmDialogIsOpen(false)}
                     onConfirm={async () => {
                         await customer.logout()
-                        await basket.getOrCreateBasket()
                         history.replace('/')
+                        navigate('/login')
```

Comment:
gotta remove the line above! It's trying to navigate to home page!

```diff
- history.replace('/')
```

Reply:
Should redirect to homepage make more sense than directing to login page? 🤔 , say, a user logs out at checkout, directing them to home page means to start the shopping flow from home page.